boards/nxp/mimxrt700_evk/doc/index.rst

```diff
@@ -0,0 +1,192 @@
+.. zephyr:board:: mimxrt700_evk
+
+Overview
+********
+
+The new i.MX RT700 CPU architecture is composed of a high-performance main compute subsystem,
+a secondary “always-on” sense-compute subsystem and specialized coprocessors.
+
+The main compute subsystem uses a 325 MHz capable Arm® Cortex®-M33 (CM33).
+Similar to the i.MX RT600 crossover MCU, the i.MX RT700 includes a Cadence Tensilica® HiFi 4 DSP.
+The HiFi 4 is a high performance DSP core based upon a Very Long Instruction Word (VLIW) architecture,
+which is capable of processing up to eight 32x16 MACs per instruction cycle. It can be used for offloading
+high-performance numerical tasks such as audio and image processing and supports both fixed-point and
+floating-point operations.
+
+Hardware
+********
+
+- Main Compute Subsystem:
+   - Arm Cortex-M33 up to 325 MHz
+   - HiFi 4 DSP up to 325 MHz
+   - eIQ Neutron NPU up to 325 MHz
+- Sense Compute Subsystem:
+   - Arm Cortex-M33 up to 250 MHz
+   - HiFi 1 DSP up to 250 MHz
```

Comment:
As per my earlier comment this should be:

```suggestion
- Main Compute Subsystem:

  - Arm Cortex-M33 up to 325 MHz
  - HiFi 4 DSP up to 325 MHz
  - eIQ Neutron NPU up to 325 MHz
- Sense Compute Subsystem:

  - Arm Cortex-M33 up to 250 MHz
  - HiFi 1 DSP up to 250 MHz
```

Reply:
Sorry, I will check the generated doc online and update it together. Thank you.

Reply:
May I ask why we need to empty line?  I checked the doc generated, and I think there is no issue on generated doc：
![Uploading image.png…]()

Reply:
because what you're seeing here is a "definition list" (see https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#definition-lists). Not how "Main Compute subsytem" and "Sense Compute Subsytem" are highlighted and how they are styled differently as the other items of the same level ("7.5 MB on-chip SRAM", etc.)
Adding the empty line and having correct alignment of the sub-list as per my suggested change ensures this is properly formatted as a bullet/sub-bullet list (see https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists)

Reply:
Thank you, I have updated.